requirements.in

```diff
@@ -1,5 +1,6 @@
 bs4
 cloudflare
+croniter
```

Comment:
Did you want to use this for something? (Not seeing it referenced in case it's used in an unobvious place.)

Reply:
This is required by django-q2 to parse cron expressions. https://django-q2.readthedocs.io/en/master/schedules.html#django_q.Schedule.cron